repos/system_upgrade/el7toel8/models/dnfplugininfo.py

```diff
@@ -0,0 +1,16 @@
+from leapp.models import Model, fields
+from leapp.topics import SystemInfoTopic
+
+
+class DNFPluginInfo(Model):
+    """
+    Represents information about what DNF plugin and in which stage
+    to disable.
+
+    Available stages: "check", "download" and "upgrade
+    """
```

Comment:
The use model of the model is not following the name and docstring. When the message is consumed, only the first one is processed. The others are ignored. The model should be renamed and documented properly or the use should be reimplemented.

Reply:
I see, I forgot to push one of my changes (which consumes all messages). This PR was open mainly for a concept review but anyway what name would you suggest?

Reply:
@pirat89 pls comment in regards of the above...

Reply:
I am not good in creating names. @bocekm is definitely better in that. I would use something like `DNFPluginTask` instead. But it doesn't seem to be good ideat. You can keep it or Michal can come up with something better.

Reply:
@pirat89 I like "DNFPluginTask" actually :)